### PR TITLE
Use Airflow trademark as it has been registered

### PR DESCRIPTION
**Pull request by pankajastro**
## Description

Replace "Apache Airflow" with "Apache Airflow®"

Airflow Dev mailing list: https://lists.apache.org/thread/0fmp9olljfkm3vmg3ns5qd1o95sm7235#:~:text=Hello%20Airflow%20community%2C%20The%20PMC,later%2C%20ASF%20got%20the%20trademark.

### CODE REVIEW

File: docs/getting_started/docker.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ Step-by-step instructions
 
 **Install Airflow and Cosmos**
 
-Create a python virtualenv, activate it, upgrade pip to the latest version and install apache airflow & astronomer-postgres
+Create a python virtualenv, activate it, upgrade pip to the latest version and install apache airflow® & astronomer-postgres
```
Comment (pankajkoti) [inline docs/getting_started/docker.rst:25]:
```suggestion
Create a python virtualenv, activate it, upgrade pip to the latest version and install Apache Airflow® & astronomer-postgres
```

**Review by pankajkoti — COMMENTED** [review]
Is it also a requirement that we need to add a link back to the Apache Airflow page?

**Comment by pankajastro** [comment]
> Is it also a requirement that we need to add a link back to the Apache Airflow page?

Yes, it is required. Thank you!

**Review by tatiana — COMMENTED** [review]
@pankajastro I'm unsure about this change, I think there is a high likelihood we'll forget adding this symbol after every mention to Apache Airflow.

Also, dbt is equally registered as a trademark: https://www.getdbt.com/assets/docs/dbtTrademarkGuidelines.pdf

And probably other technologies we're mentioning in the docs as well, including Databricks and Snowflake. Is this change really necessary?

**Comment by pankajastro** [comment]
> @pankajastro I'm unsure about this change, I think there is a high likelihood we'll forget adding this symbol after every mention to Apache Airflow.
> 
> Also, dbt is equally registered as a trademark: https://www.getdbt.com/assets/docs/dbtTrademarkGuidelines.pdf
> 
> And probably other technologies we're mentioning in the docs as well, including Databricks and Snowflake. Is this change really necessary?

I checked the dbt trademark guidelines, and the documentation advises using `dbt™`. Does this mean it is not a registered trademark? 
I couldn't find any sources for Databricks and Snowflake that provide guidelines on how to use their names correctly.

**Comment by tatiana** [comment]
> > @pankajastro I'm unsure about this change, I think there is a high likelihood we'll forget adding this symbol after every mention to Apache Airflow.
> > Also, dbt is equally registered as a trademark: https://www.getdbt.com/assets/docs/dbtTrademarkGuidelines.pdf
> > And probably other technologies we're mentioning in the docs as well, including Databricks and Snowflake. Is this change really necessary?
> 
> I checked the dbt trademark guidelines, and the documentation advises using `dbt™`. Does this mean it is not a registered trademark? I couldn't find any sources for Databricks and Snowflake that provide guidelines on how to use their names correctly.

@pankajastro Fair enough, let's make our docs compliant with Apache Airflow® - this is the scope of this PR - and when we learn more about the other projects, we can update the docs. Thanks a lot for checking this, @pankajastro !

**Review by tatiana — APPROVED** [review]
> > @pankajastro I'm unsure about this change, I think there is a high likelihood we'll forget adding this symbol after every mention to Apache Airflow.
> > Also, dbt is equally registered as a trademark: https://www.getdbt.com/assets/docs/dbtTrademarkGuidelines.pdf
> > And probably other technologies we're mentioning in the docs as well, including Databricks and Snowflake. Is this change really necessary?
> 
> I checked the dbt trademark guidelines, and the documentation advises using `dbt™`. Does this mean it is not a registered trademark? I couldn't find any sources for Databricks and Snowflake that provide guidelines on how to use their names correctly.

@pankajastro Fair enough, let's make our docs compliant with Apache Airflow® - this is the scope of this PR - and when we learn more about the other projects, we can update the docs. Thanks a lot for checking this, @pankajastro !

**Pull request merged**